sensitivity-api/src/main/java/com/powsybl/sensitivity/json/SensitivityAnalysisParametersSerializer.java

```diff
@@ -11,32 +11,32 @@
 import com.fasterxml.jackson.databind.ser.std.StdSerializer;
 import com.powsybl.commons.json.JsonUtil;
 import com.powsybl.loadflow.json.JsonLoadFlowParameters;
-import com.powsybl.sensitivity.SensitivityComputationParameters;
+import com.powsybl.sensitivity.SensitivityAnalysisParameters;
 
 import java.io.IOException;
 
 /**
- * Json serializer for sensitivity computation parameters
+ * Json serializer for sensitivity analysis parameters
  *
  * @author Sebastien Murgey <sebastien.murgey at rte-france.com>
  */
-public class SensitivityComputationParametersSerializer extends StdSerializer<SensitivityComputationParameters> {
+public class SensitivityAnalysisParametersSerializer extends StdSerializer<SensitivityAnalysisParameters> {
 
-    SensitivityComputationParametersSerializer() {
-        super(SensitivityComputationParameters.class);
+    SensitivityAnalysisParametersSerializer() {
```

Comment:
Question: should we make this constructor public?

Reply:
In the use, it is more supposed to be added to the `SensitivityAnalysisParametersJsonModule`. I think there is no "security" issue by making it public but I am not sure it will ever be used.